### ISSUE TITLE
Errors when Attempting to build Deb on Trusty Tahr 14.04.

### ISSUE THREAD

**Issue by Austcool-Walker** [variar/klogg#239]
```bash
Scanning dependencies of target generate_version
fatal: Not a git repository (or any of the parent directories): .git
Git branch: 
fatal: Not a git repository (or any of the parent directories): .git
Git commit: 
fatal: Not a git repository (or any of the parent directories): .git
Git describe: 
[  0%] Built target generate_version
Scanning dependencies of target absl_log_severity
[  0%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_log_severity.dir/log_severity.cc.o
[  0%] Linking CXX static library ../../../../output/libabsl_log_severity.a
[  0%] Built target absl_log_severity
Scanning dependencies of target absl_raw_logging_internal
[  0%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_raw_logging_internal.dir/internal/raw_logging.cc.o
[  1%] Linking CXX static library ../../../../output/libabsl_raw_logging_internal.a
[  1%] Built target absl_raw_logging_internal
Scanning dependencies of target absl_dynamic_annotations
[  1%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_dynamic_annotations.dir/dynamic_annotations.cc.o
[  1%] Linking CXX static library ../../../../output/libabsl_dynamic_annotations.a
[  1%] Built target absl_dynamic_annotations
Scanning dependencies of target absl_spinlock_wait
[  1%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_spinlock_wait.dir/internal/spinlock_wait.cc.o
[  1%] Linking CXX static library ../../../../output/libabsl_spinlock_wait.a
[  1%] Built target absl_spinlock_wait
Scanning dependencies of target absl_base
[  1%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_base.dir/internal/cycleclock.cc.o
[  2%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_base.dir/internal/spinlock.cc.o
[  2%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_base.dir/internal/sysinfo.cc.o
[  2%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_base.dir/internal/thread_identity.cc.o
[  2%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_base.dir/internal/unscaledcycleclock.cc.o
[  2%] Linking CXX static library ../../../../output/libabsl_base.a
[  2%] Built target absl_base
Scanning dependencies of target absl_malloc_internal
[  3%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_malloc_internal.dir/internal/low_level_alloc.cc.o
[  3%] Linking CXX static library ../../../../output/libabsl_malloc_internal.a
[  3%] Built target absl_malloc_internal
Scanning dependencies of target absl_exponential_biased
[  3%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_exponential_biased.dir/internal/exponential_biased.cc.o
[  4%] Linking CXX static library ../../../../output/libabsl_exponential_biased.a
[  4%] Built target absl_exponential_biased
Scanning dependencies of target absl_periodic_sampler
[  4%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_periodic_sampler.dir/internal/periodic_sampler.cc.o
[  4%] Linking CXX static library ../../../../output/libabsl_periodic_sampler.a
[  4%] Built target absl_periodic_sampler
Scanning dependencies of target absl_throw_delegate
[  4%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_throw_delegate.dir/internal/throw_delegate.cc.o
[  4%] Linking CXX static library ../../../../output/libabsl_throw_delegate.a
[  4%] Built target absl_throw_delegate
Scanning dependencies of target absl_scoped_set_env
[  4%] Building CXX object 3rdparty/abseil/absl/base/CMakeFiles/absl_scoped_set_env.dir/internal/scoped_set_env.cc.o
[  4%] Linking CXX static library ../../../../output/libabsl_scoped_set_env.a
[  4%] Built target absl_scoped_set_env
Scanning dependencies of target absl_time_zone
[  4%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_fixed.cc.o
[  4%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_format.cc.o
[  4%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_if.cc.o
[  4%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_impl.cc.o
[  4%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_info.cc.o
[  4%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_libc.cc.o
[  5%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_lookup.cc.o
[  5%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/time_zone_posix.cc.o
[  5%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time_zone.dir/internal/cctz/src/zone_info_source.cc.o
[  5%] Linking CXX static library ../../../../output/libabsl_time_zone.a
[  5%] Built target absl_time_zone
Scanning dependencies of target absl_debugging_internal
[  5%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_debugging_internal.dir/internal/address_is_readable.cc.o
[  5%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_debugging_internal.dir/internal/elf_mem_image.cc.o
[  5%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_debugging_internal.dir/internal/vdso_support.cc.o
[  5%] Linking CXX static library ../../../../output/libabsl_debugging_internal.a
[  5%] Built target absl_debugging_internal
Scanning dependencies of target absl_stacktrace
[  5%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_stacktrace.dir/stacktrace.cc.o
[  5%] Linking CXX static library ../../../../output/libabsl_stacktrace.a
[  5%] Built target absl_stacktrace
Scanning dependencies of target absl_demangle_internal
[  6%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_demangle_internal.dir/internal/demangle.cc.o
[  6%] Linking CXX static library ../../../../output/libabsl_demangle_internal.a
[  6%] Built target absl_demangle_internal
Scanning dependencies of target absl_symbolize
[  6%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_symbolize.dir/symbolize.cc.o
[  6%] Linking CXX static library ../../../../output/libabsl_symbolize.a
[  6%] Built target absl_symbolize
Scanning dependencies of target absl_int128
[  7%] Building CXX object 3rdparty/abseil/absl/numeric/CMakeFiles/absl_int128.dir/int128.cc.o
[  7%] Linking CXX static library ../../../../output/libabsl_int128.a
[  7%] Built target absl_int128
Scanning dependencies of target absl_strings_internal
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings_internal.dir/internal/ostringstream.cc.o
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings_internal.dir/internal/utf8.cc.o
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings_internal.dir/internal/escaping.cc.o
[  7%] Linking CXX static library ../../../../output/libabsl_strings_internal.a
[  7%] Built target absl_strings_internal
Scanning dependencies of target absl_strings
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/ascii.cc.o
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/charconv.cc.o
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/escaping.cc.o
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/internal/charconv_bigint.cc.o
[  7%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/internal/charconv_parse.cc.o
[  8%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/internal/memutil.cc.o
[  8%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/match.cc.o
[  8%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/numbers.cc.o
[  8%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/str_cat.cc.o
[  8%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/str_replace.cc.o
[  8%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/str_split.cc.o
[  8%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/string_view.cc.o
[  9%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_strings.dir/substitute.cc.o
[  9%] Linking CXX static library ../../../../output/libabsl_strings.a
[  9%] Built target absl_strings
Scanning dependencies of target absl_graphcycles_internal
[ 10%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_graphcycles_internal.dir/internal/graphcycles.cc.o
[ 10%] Linking CXX static library ../../../../output/libabsl_graphcycles_internal.a
[ 10%] Built target absl_graphcycles_internal
Scanning dependencies of target absl_civil_time
[ 11%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_civil_time.dir/internal/cctz/src/civil_time_detail.cc.o
[ 11%] Linking CXX static library ../../../../output/libabsl_civil_time.a
[ 11%] Built target absl_civil_time
Scanning dependencies of target absl_time
[ 11%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time.dir/civil_time.cc.o
[ 11%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time.dir/clock.cc.o
[ 11%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time.dir/duration.cc.o
[ 11%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time.dir/format.cc.o
[ 12%] Building CXX object 3rdparty/abseil/absl/time/CMakeFiles/absl_time.dir/time.cc.o
[ 12%] Linking CXX static library ../../../../output/libabsl_time.a
[ 12%] Built target absl_time
Scanning dependencies of target absl_synchronization
[ 13%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_synchronization.dir/barrier.cc.o
[ 13%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_synchronization.dir/blocking_counter.cc.o
[ 13%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_synchronization.dir/internal/create_thread_identity.cc.o
[ 13%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_synchronization.dir/internal/per_thread_sem.cc.o
[ 13%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_synchronization.dir/internal/waiter.cc.o
[ 13%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_synchronization.dir/notification.cc.o
[ 13%] Building CXX object 3rdparty/abseil/absl/synchronization/CMakeFiles/absl_synchronization.dir/mutex.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_synchronization.a
[ 14%] Built target absl_synchronization
Scanning dependencies of target absl_hashtablez_sampler
[ 14%] Building CXX object 3rdparty/abseil/absl/container/CMakeFiles/absl_hashtablez_sampler.dir/internal/hashtablez_sampler.cc.o
[ 14%] Building CXX object 3rdparty/abseil/absl/container/CMakeFiles/absl_hashtablez_sampler.dir/internal/hashtablez_sampler_force_weak_definition.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_hashtablez_sampler.a
[ 14%] Built target absl_hashtablez_sampler
Scanning dependencies of target absl_bad_optional_access
[ 14%] Building CXX object 3rdparty/abseil/absl/types/CMakeFiles/absl_bad_optional_access.dir/bad_optional_access.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_bad_optional_access.a
[ 14%] Built target absl_bad_optional_access
Scanning dependencies of target absl_raw_hash_set
[ 14%] Building CXX object 3rdparty/abseil/absl/container/CMakeFiles/absl_raw_hash_set.dir/internal/raw_hash_set.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_raw_hash_set.a
[ 14%] Built target absl_raw_hash_set
Scanning dependencies of target absl_examine_stack
[ 14%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_examine_stack.dir/internal/examine_stack.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_examine_stack.a
[ 14%] Built target absl_examine_stack
Scanning dependencies of target absl_leak_check_disable
[ 14%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_leak_check_disable.dir/leak_check_disable.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_leak_check_disable.a
[ 14%] Built target absl_leak_check_disable
Scanning dependencies of target absl_failure_signal_handler
[ 14%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_failure_signal_handler.dir/failure_signal_handler.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_failure_signal_handler.a
[ 14%] Built target absl_failure_signal_handler
Scanning dependencies of target absl_leak_check
[ 14%] Building CXX object 3rdparty/abseil/absl/debugging/CMakeFiles/absl_leak_check.dir/leak_check.cc.o
[ 14%] Linking CXX static library ../../../../output/libabsl_leak_check.a
[ 14%] Built target absl_leak_check
Scanning dependencies of target absl_str_format_internal
[ 14%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_str_format_internal.dir/internal/str_format/arg.cc.o
[ 14%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_str_format_internal.dir/internal/str_format/bind.cc.o
[ 14%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_str_format_internal.dir/internal/str_format/extension.cc.o
[ 14%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_str_format_internal.dir/internal/str_format/float_conversion.cc.o
[ 14%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_str_format_internal.dir/internal/str_format/output.cc.o
[ 15%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_str_format_internal.dir/internal/str_format/parser.cc.o
[ 15%] Linking CXX static library ../../../../output/libabsl_str_format_internal.a
[ 15%] Built target absl_str_format_internal
Scanning dependencies of target absl_flags_marshalling
[ 15%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_marshalling.dir/marshalling.cc.o
[ 15%] Linking CXX static library ../../../../output/libabsl_flags_marshalling.a
[ 15%] Built target absl_flags_marshalling
Scanning dependencies of target absl_flags_program_name
[ 15%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_program_name.dir/internal/program_name.cc.o
[ 15%] Linking CXX static library ../../../../output/libabsl_flags_program_name.a
[ 15%] Built target absl_flags_program_name
Scanning dependencies of target absl_flags_config
[ 15%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_config.dir/usage_config.cc.o
[ 15%] Linking CXX static library ../../../../output/libabsl_flags_config.a
[ 15%] Built target absl_flags_config
Scanning dependencies of target absl_flags_registry
[ 16%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_registry.dir/internal/registry.cc.o
[ 16%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_registry.dir/internal/type_erased.cc.o
[ 16%] Linking CXX static library ../../../../output/libabsl_flags_registry.a
[ 16%] Built target absl_flags_registry
Scanning dependencies of target absl_flags_internal
[ 17%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_internal.dir/internal/flag.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_flags_internal.a
[ 17%] Built target absl_flags_internal
Scanning dependencies of target absl_flags
[ 17%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags.dir/flag.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_flags.a
[ 17%] Built target absl_flags
Scanning dependencies of target absl_flags_usage_internal
[ 17%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_usage_internal.dir/internal/usage.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_flags_usage_internal.a
[ 17%] Built target absl_flags_usage_internal
Scanning dependencies of target absl_flags_usage
[ 17%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_usage.dir/usage.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_flags_usage.a
[ 17%] Built target absl_flags_usage
Scanning dependencies of target absl_flags_parse
[ 17%] Building CXX object 3rdparty/abseil/absl/flags/CMakeFiles/absl_flags_parse.dir/parse.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_flags_parse.a
[ 17%] Built target absl_flags_parse
Scanning dependencies of target absl_city
[ 17%] Building CXX object 3rdparty/abseil/absl/hash/CMakeFiles/absl_city.dir/internal/city.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_city.a
[ 17%] Built target absl_city
Scanning dependencies of target absl_bad_variant_access
[ 17%] Building CXX object 3rdparty/abseil/absl/types/CMakeFiles/absl_bad_variant_access.dir/bad_variant_access.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_bad_variant_access.a
[ 17%] Built target absl_bad_variant_access
Scanning dependencies of target absl_hash
[ 17%] Building CXX object 3rdparty/abseil/absl/hash/CMakeFiles/absl_hash.dir/internal/hash.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_hash.a
[ 17%] Built target absl_hash
Scanning dependencies of target absl_random_internal_seed_material
[ 17%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_seed_material.dir/internal/seed_material.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_random_internal_seed_material.a
[ 17%] Built target absl_random_internal_seed_material
Scanning dependencies of target absl_random_internal_randen_hwaes_impl
[ 17%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_randen_hwaes_impl.dir/internal/randen_hwaes.cc.o
[ 17%] Linking CXX static library ../../../../output/libabsl_random_internal_randen_hwaes_impl.a
[ 17%] Built target absl_random_internal_randen_hwaes_impl
Scanning dependencies of target absl_random_internal_randen_hwaes
[ 18%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_randen_hwaes.dir/internal/randen_detect.cc.o
[ 18%] Linking CXX static library ../../../../output/libabsl_random_internal_randen_hwaes.a
[ 18%] Built target absl_random_internal_randen_hwaes
Scanning dependencies of target absl_random_internal_randen_slow
[ 18%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_randen_slow.dir/internal/randen_slow.cc.o
[ 18%] Linking CXX static library ../../../../output/libabsl_random_internal_randen_slow.a
[ 18%] Built target absl_random_internal_randen_slow
Scanning dependencies of target absl_random_internal_randen
[ 18%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_randen.dir/internal/randen.cc.o
[ 18%] Linking CXX static library ../../../../output/libabsl_random_internal_randen.a
[ 18%] Built target absl_random_internal_randen
Scanning dependencies of target absl_random_internal_distribution_test_util
[ 19%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_distribution_test_util.dir/internal/chi_square.cc.o
[ 19%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_distribution_test_util.dir/internal/distribution_test_util.cc.o
[ 19%] Linking CXX static library ../../../../output/libabsl_random_internal_distribution_test_util.a
[ 19%] Built target absl_random_internal_distribution_test_util
Scanning dependencies of target absl_random_seed_gen_exception
[ 20%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_seed_gen_exception.dir/seed_gen_exception.cc.o
[ 20%] Linking CXX static library ../../../../output/libabsl_random_seed_gen_exception.a
[ 20%] Built target absl_random_seed_gen_exception
Scanning dependencies of target absl_random_internal_pool_urbg
[ 20%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_internal_pool_urbg.dir/internal/pool_urbg.cc.o
[ 20%] Linking CXX static library ../../../../output/libabsl_random_internal_pool_urbg.a
[ 20%] Built target absl_random_internal_pool_urbg
Scanning dependencies of target absl_random_seed_sequences
[ 20%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_seed_sequences.dir/seed_sequences.cc.o
[ 20%] Linking CXX static library ../../../../output/libabsl_random_seed_sequences.a
[ 20%] Built target absl_random_seed_sequences
Scanning dependencies of target absl_random_distributions
[ 20%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_distributions.dir/discrete_distribution.cc.o
[ 20%] Building CXX object 3rdparty/abseil/absl/random/CMakeFiles/absl_random_distributions.dir/gaussian_distribution.cc.o
[ 20%] Linking CXX static library ../../../../output/libabsl_random_distributions.a
[ 20%] Built target absl_random_distributions
Scanning dependencies of target absl_cord
[ 20%] Building CXX object 3rdparty/abseil/absl/strings/CMakeFiles/absl_cord.dir/cord.cc.o
[ 20%] Linking CXX static library ../../../../output/libabsl_cord.a
[ 20%] Built target absl_cord
Scanning dependencies of target absl_status
[ 21%] Building CXX object 3rdparty/abseil/absl/status/CMakeFiles/absl_status.dir/status.cc.o
[ 21%] Building CXX object 3rdparty/abseil/absl/status/CMakeFiles/absl_status.dir/status_payload_printer.cc.o
[ 21%] Linking CXX static library ../../../../output/libabsl_status.a
[ 21%] Built target absl_status
Scanning dependencies of target absl_bad_any_cast_impl
[ 21%] Building CXX object 3rdparty/abseil/absl/types/CMakeFiles/absl_bad_any_cast_impl.dir/bad_any_cast.cc.o
[ 21%] Linking CXX static library ../../../../output/libabsl_bad_any_cast_impl.a
[ 21%] Built target absl_bad_any_cast_impl
Scanning dependencies of target uchardet
[ 21%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/CharDistribution.cpp.o
[ 21%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/JpCntx.cpp.o
[ 21%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangArabicModel.cpp.o
[ 22%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangBulgarianModel.cpp.o
[ 22%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangRussianModel.cpp.o
[ 22%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangEsperantoModel.cpp.o
[ 22%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangFrenchModel.cpp.o
[ 22%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangDanishModel.cpp.o
[ 22%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangGermanModel.cpp.o
[ 22%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangGreekModel.cpp.o
[ 23%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangHungarianModel.cpp.o
[ 23%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangHebrewModel.cpp.o
[ 23%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangSpanishModel.cpp.o
[ 23%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangThaiModel.cpp.o
[ 23%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangTurkishModel.cpp.o
[ 23%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangVietnameseModel.cpp.o
[ 23%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangCroatianModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangCzechModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangEstonianModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangFinnishModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangIrishModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangItalianModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangLatvianModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangLithuanianModel.cpp.o
[ 24%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangMalteseModel.cpp.o
[ 25%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangPolishModel.cpp.o
[ 25%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangPortugueseModel.cpp.o
[ 25%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangRomanianModel.cpp.o
[ 25%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangSlovakModel.cpp.o
[ 25%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangSloveneModel.cpp.o
[ 25%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/LangModels/LangSwedishModel.cpp.o
[ 25%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsHebrewProber.cpp.o
[ 26%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsCharSetProber.cpp.o
[ 26%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsBig5Prober.cpp.o
[ 26%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsEUCJPProber.cpp.o
[ 26%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsEUCKRProber.cpp.o
[ 26%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsEUCTWProber.cpp.o
[ 26%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsEscCharsetProber.cpp.o
[ 26%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsEscSM.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsGB2312Prober.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsMBCSGroupProber.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsMBCSSM.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsSBCSGroupProber.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsSBCharSetProber.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsSJISProber.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsUTF8Prober.cpp.o
[ 27%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsLatin1Prober.cpp.o
[ 28%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/nsUniversalDetector.cpp.o
[ 28%] Building CXX object 3rdparty/uchardet/CMakeFiles/uchardet.dir/src/uchardet.cpp.o
[ 28%] Linking CXX static library ../../output/libuchardet.a
[ 28%] Built target uchardet
Scanning dependencies of target efsw
[ 28%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/Debug.cpp.o
[ 29%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/DirectorySnapshot.cpp.o
[ 29%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/DirectorySnapshotDiff.cpp.o
[ 29%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/DirWatcherGeneric.cpp.o
/home/noire/Downloads/klogg-continuous-linux/3rdparty/efsw/src/efsw/DirWatcherGeneric.cpp:91:63: warning: ISO C99 requires rest arguments to be used [enabled by default]
     efDEBUG( "resetDirectory(): Parent is NULL. Fatal error." );
                                                               ^
cc1plus: warning: unrecognized command line option "-Wno-implicit-fallthrough" [enabled by default]
[ 29%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/FileInfo.cpp.o
[ 29%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/FileSystem.cpp.o
[ 29%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/FileWatcher.cpp.o
/home/noire/Downloads/klogg-continuous-linux/3rdparty/efsw/src/efsw/FileWatcher.cpp:55:45: warning: ISO C99 requires rest arguments to be used [enabled by default]
         efDEBUG( "Using backend: Generic\n" );
                                             ^
cc1plus: warning: unrecognized command line option "-Wno-implicit-fallthrough" [enabled by default]
[ 29%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/FileWatcherCWrapper.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/FileWatcherGeneric.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/FileWatcherImpl.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/Log.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/Mutex.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/String.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/System.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/Thread.cpp.o
[ 30%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/Watcher.cpp.o
[ 31%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/WatcherGeneric.cpp.o
[ 31%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/platform/posix/FileSystemImpl.cpp.o
[ 31%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/platform/posix/MutexImpl.cpp.o
[ 31%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/platform/posix/SystemImpl.cpp.o
[ 31%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/platform/posix/ThreadImpl.cpp.o
/home/noire/Downloads/klogg-continuous-linux/3rdparty/efsw/src/efsw/platform/posix/ThreadImpl.cpp:19:40: warning: ISO C99 requires rest arguments to be used [enabled by default]
   efDEBUG( "Failed to create thread\n" );
                                        ^
cc1plus: warning: unrecognized command line option "-Wno-implicit-fallthrough" [enabled by default]
[ 31%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/FileWatcherInotify.cpp.o
[ 31%] Building CXX object 3rdparty/efsw/CMakeFiles/efsw.dir/src/efsw/WatcherInotify.cpp.o
[ 32%] Linking CXX static library ../../output/libefsw.a
[ 32%] Built target efsw
Scanning dependencies of target tbb_static
[ 32%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/old/concurrent_queue_v2.cpp.o
[ 32%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/old/concurrent_vector_v2.cpp.o
[ 32%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/old/spin_rw_mutex_v2.cpp.o
[ 32%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/old/task_v2.cpp.o
[ 33%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/arena.cpp.o
[ 33%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/cache_aligned_allocator.cpp.o
[ 33%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/concurrent_hash_map.cpp.o
[ 33%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/concurrent_monitor.cpp.o
[ 33%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/concurrent_queue.cpp.o
[ 33%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/concurrent_vector.cpp.o
[ 33%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/condition_variable.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/critical_section.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/dynamic_link.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/governor.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/itt_notify.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/market.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/mutex.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/observer_proxy.cpp.o
[ 34%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/pipeline.cpp.o
[ 35%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/private_server.cpp.o
[ 35%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/queuing_mutex.cpp.o
[ 35%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/queuing_rw_mutex.cpp.o
[ 35%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/reader_writer_lock.cpp.o
[ 35%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/recursive_mutex.cpp.o
[ 35%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/scheduler.cpp.o
[ 35%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/semaphore.cpp.o
[ 36%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/spin_mutex.cpp.o
[ 36%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/spin_rw_mutex.cpp.o
[ 36%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/task.cpp.o
[ 36%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/task_group_context.cpp.o
[ 36%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/tbb_main.cpp.o
[ 36%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/tbb_misc.cpp.o
[ 36%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/tbb_misc_ex.cpp.o
[ 37%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/tbb_statistics.cpp.o
[ 37%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/tbb_thread.cpp.o
[ 37%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/tbb/x86_rtm_rw_mutex.cpp.o
[ 37%] Building CXX object 3rdparty/tbb/CMakeFiles/tbb_static.dir/src/rml/client/rml_tbb.cpp.o
[ 37%] Linking CXX static library ../../output/libtbb_static.a
[ 37%] Built target tbb_static
Scanning dependencies of target tbb_def_files
[ 37%] Preprocessing tbb.def
[ 37%] Preprocessing tbbmalloc.def
[ 37%] Built target tbb_def_files
Scanning dependencies of target klogg_tbbmalloc
[ 37%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc.dir/src/tbbmalloc/backend.cpp.o
[ 37%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc.dir/src/tbbmalloc/large_objects.cpp.o
[ 38%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc.dir/src/tbbmalloc/backref.cpp.o
[ 38%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc.dir/src/tbbmalloc/tbbmalloc.cpp.o
[ 38%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc.dir/src/tbbmalloc/frontend.cpp.o
[ 38%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc.dir/src/tbb/itt_notify.cpp.o
[ 38%] Linking CXX shared library ../../output/libklogg_tbbmalloc.so
[ 38%] Built target klogg_tbbmalloc
Scanning dependencies of target klogg_tbbmalloc_proxy
[ 38%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc_proxy.dir/src/tbbmalloc/proxy.cpp.o
[ 38%] Building CXX object 3rdparty/tbb/CMakeFiles/klogg_tbbmalloc_proxy.dir/src/tbbmalloc/tbb_function_replacement.cpp.o
[ 39%] Linking CXX shared library ../../output/libklogg_tbbmalloc_proxy.so
[ 39%] Built target klogg_tbbmalloc_proxy
Scanning dependencies of target singleapp_autogen
[ 40%] Automatic MOC for target singleapp
[ 40%] Built target singleapp_autogen
Scanning dependencies of target singleapp
[ 40%] Building CXX object 3rdparty/singleapp/CMakeFiles/singleapp.dir/singleapp_autogen/mocs_compilation.cpp.o
[ 40%] Building CXX object 3rdparty/singleapp/CMakeFiles/singleapp.dir/src/singleapplication.cpp.o
[ 40%] Building CXX object 3rdparty/singleapp/CMakeFiles/singleapp.dir/src/singleapplication_p.cpp.o
[ 40%] Linking CXX static library ../../output/libsingleapp.a
[ 40%] Built target singleapp
Scanning dependencies of target cmzlib
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/adler32.c.o
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/compress.c.o
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/crc32.c.o
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/deflate.c.o
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/gzio.c.o
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/inffast.c.o
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/inflate.c.o
[ 41%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/inftrees.c.o
[ 42%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/trees.c.o
[ 42%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/uncompr.c.o
[ 42%] Building C object 3rdparty/cmzlib/CMakeFiles/cmzlib.dir/zutil.c.o
[ 42%] Linking C static library ../../output/libcmzlib.a
[ 42%] Built target cmzlib
Scanning dependencies of target cmbzip2
[ 42%] Building C object 3rdparty/cmbzip2/CMakeFiles/cmbzip2.dir/blocksort.c.o
[ 42%] Building C object 3rdparty/cmbzip2/CMakeFiles/cmbzip2.dir/huffman.c.o
[ 42%] Building C object 3rdparty/cmbzip2/CMakeFiles/cmbzip2.dir/crctable.c.o
[ 43%] Building C object 3rdparty/cmbzip2/CMakeFiles/cmbzip2.dir/randtable.c.o
[ 43%] Building C object 3rdparty/cmbzip2/CMakeFiles/cmbzip2.dir/compress.c.o
[ 43%] Building C object 3rdparty/cmbzip2/CMakeFiles/cmbzip2.dir/decompress.c.o
[ 43%] Building C object 3rdparty/cmbzip2/CMakeFiles/cmbzip2.dir/bzlib.c.o
[ 43%] Linking C static library ../../output/libcmbzip2.a
[ 43%] Built target cmbzip2
Scanning dependencies of target cmliblzma
[ 43%] Building C object 3rdparty/cmliblzma/CMakeFiles/cmliblzma.dir/liblzma/check/check.c.o
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.c:1:1: warning: C++ style comments are not allowed in ISO C90 [enabled by default]
 ///////////////////////////////////////////////////////////////////////////////
 ^
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.c:1:1: warning: (this will be reported only once per input file) [enabled by default]
In file included from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.c:13:0:
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.h:1:1: warning: C++ style comments are not allowed in ISO C90 [enabled by default]
 ///////////////////////////////////////////////////////////////////////////////
 ^
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.h:1:1: warning: (this will be reported only once per input file) [enabled by default]
In file included from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.h:16:0,
                 from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.c:13:
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/common/common.h:1:1: warning: C++ style comments are not allowed in ISO C90 [enabled by default]
 ///////////////////////////////////////////////////////////////////////////////
 ^
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/common/common.h:1:1: warning: (this will be reported only once per input file) [enabled by default]
In file included from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/common/common.h:16:0,
                 from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.h:16,
                 from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.c:13:
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/common/sysdefs.h:1:1: warning: C++ style comments are not allowed in ISO C90 [enabled by default]
 ///////////////////////////////////////////////////////////////////////////////
 ^
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/common/sysdefs.h:1:1: warning: (this will be reported only once per input file) [enabled by default]
In file included from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.h:16:0,
                 from /home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/check/check.c:13:
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/common/common.h:106:27: error: expected ‘;’, ‘,’ or ‘)’ before ‘in’
   const uint8_t *restrict in, size_t *restrict in_pos,
                           ^
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/common/common.h:149:2: error: unknown type name ‘lzma_code_function’
  lzma_code_function code;
  ^
/home/noire/Downloads/klogg-continuous-linux/3rdparty/cmliblzma/liblzma/common/common.h:267:51: error: expected ‘;’, ‘,’ or ‘)’ before ‘in’
 extern size_t lzma_bufcpy(const uint8_t *restrict in, size_t *restrict in_pos,
                                                   ^
cc1: warning: unrecognized command line option "-Wno-implicit-fallthrough" [enabled by default]
make[2]: *** [3rdparty/cmliblzma/CMakeFiles/cmliblzma.dir/liblzma/check/check.c.o] Error 1
make[1]: *** [3rdparty/cmliblzma/CMakeFiles/cmliblzma.dir/all] Error 2
make: *** [all] Error 2
```
[output-trusty-tahr.log](https://github.com/variar/klogg/files/5790009/output-trusty-tahr.log)
[CMakeLists.txt](https://github.com/variar/klogg/files/5790015/CMakeLists.txt)

**Comment by variar** [comment]
What version of c/c++ compiler is being used? Can't find it from logs.

**Comment by Austcool-Walker** [comment]
> c/c++ compile

Terminal Returns.
```$ /usr/bin/gcc -v
Using built-in specs.
COLLECT_GCC=/usr/bin/gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/4.8/lto-wrapper
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 4.8.4-2ubuntu1~14.04.4' --with-bugurl=file:///usr/share/doc/gcc-4.8/README.Bugs --enable-languages=c,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.8 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.8 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --disable-libmudflap --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-4.8-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.4)```

**Comment by variar** [comment]
That compiler is too old. GCC 5 could work, but it is not tested. GCC 8 is used in CI builds.

**Comment by Austcool-Walker** [comment]
Ok will strap homebrew then build with GCC5?

**Comment by Austcool-Walker** [comment]
Nope Still failed...
```
/home/noire/Documents/klogg/3rdparty/karchive/src/karchive.cpp: In static member function ‘static QDateTime KArchivePrivate::time_tToDateTime(uint)’:
/home/noire/Documents/klogg/3rdparty/karchive/src/karchive.cpp:662:12: error: ‘fromSecsSinceEpoch’ is not a member of ‘QDateTime’
     return QDateTime::fromSecsSinceEpoch(time_t);
            ^
At global scope:
cc1plus: warning: unrecognized command line option ‘-Wno-implicit-fallthrough’
make[2]: *** [3rdparty/karchive/CMakeFiles/klogg_karchive.dir/src/karchive.cpp.o] Error 1
make[1]: *** [3rdparty/karchive/CMakeFiles/klogg_karchive.dir/all] Error 2
make: *** [all] Error 2
cmake --build .  284.69s user 19.48s system 96% cpu 5:14.43 total
2 noire@Austcools-MacBookPro ~/Documents/klogg/build_root (git)-[master] %
```

**Comment by Austcool-Walker** [comment]
I'll build gcc8 my self if I have to

**Comment by variar** [comment]
That error is not related to gcc version. What Qt version is used? Function fromSecsSinceEpoch was introduced in Qt 5.8.  And Qt 5.9 is the oldest version that is used in CI builds.

**Comment by Austcool-Walker** [comment]
> That error is not related to gcc version. What Qt version is used? Function fromSecsSinceEpoch was introduced in Qt 5.8. And Qt 5.9 is the oldest version that is used in CI builds of klogg.

```
1 noire@Austcools-MacBookPro ~/Downloads % QT_SELECT=5 qmake -v
QMake version 3.0
Using Qt version 5.2.1 in /usr/lib/x86_64-linux-gnu
noire@Austcools-MacBookPro ~/Downloads %
```

**Comment by variar** [comment]
@Austcool-Walker any luck with that Qt/gcc combination?

**Comment by variar** [comment]
Thanks for the feedback. I've tried to build an appimage for CentOS 6 (there is a separate issue for that). However, system libraries are just too old. 

Clang is used to build klogg for Mac. And can be used on Linux based systems. I keep gcc in ci builds to make sure all three major compilers can build klogg.

**Comment by variar** [comment]
Building on older platforms will be tracked in #195
